site/pages/home/client-testimonials-slice/4/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styles from './style.css';
+
+const Four = () => {
+  const contentPart1 = 'Exciting, refreshing,';
+  const contentPart2 = 'positive & pacey.';
+  const author = '– Customer Experience Director, Retail';
+  return (
+    <div className={styles.contentWrapper}>
+      <div className={styles.centered}>
+        <div className={styles.mainWrapper}>
+          <div className={styles.main}>
+            {contentPart1}
+            <br />
+            {contentPart2}
+          </div>
```

Comment:
Ideally we wouldn't use here an extra HTML element and just restrict the size of the div so the text wraps